src/test/java/io/confluent/connect/jdbc/dialect/PostgreSqlDatabaseDialectTest.java

```diff
@@ -410,4 +415,26 @@ public void shouldSanitizeUrlWithCredentialsInUrlProperties() {
         "jdbc:postgresql://localhost/test?user=fred&password=****&ssl=true"
     );
   }
-}
\ No newline at end of file
+
+  @Test
+  @Override
+  public void bindFieldArrayUnsupported() throws SQLException {
+      // Overridden simply to dummy out the test.
```

Comment:
I think we should instead move the test out of the `BaseDialectTest` and into the dialect specific tests that this test still applies to. The test that still applies to the postgres dialect is one that tests for array of non-primitive type still throws, which should be added here.